proto/sentry_protos/sentry/v1/taskworker.proto

```diff
@@ -130,6 +132,7 @@ message SetTaskStatusRequest {
 
   // Set to true to have receive a new task in the response
   optional bool fetch_next = 4;
+  optional string fetch_next_namespace = 5;
```

Comment:
Would be nice if the grpc server could either respond with a next task if this or `fetch_next` is used.